<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpellChecker::Internal::SpellCheckerCoreOptionsWidget</class>
 <widget class="QWidget" name="SpellChecker::Internal::SpellCheckerCoreOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Spell Checker</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxSpellChecker"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="spellCheckerOptionsWidgetHolder" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
